index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    
+    <main id="main">
+        <div class="header">
+                    <H1 id="title">Dr. Norman Borlaug</H1>
```

Comment:
I think it would be better to use lowercase letter in your html tag.
Because lowercase tags improve code readability. HTML tags are typically written in lowercase, so using uppercase letters can make your code stand out and potentially distract from the actual content